src/sprints.actions.js

```diff
@@ -1,12 +1,25 @@
 const { getTeamId } = require('./team-data');
 const client = require('./jira-client');
 
+async function pageSprints(teamId, startAt = 0) {
+  const sprints = await client.makeGetRequest(`board/${ teamId }/sprint?startAt=${ startAt }`);
+  let sprintValues = sprints.values;
+
+  if (!sprints.isLast) {
+    const nextPage = await pageSprints(teamId, startAt + 50);
+    sprintValues = sprintValues.concat(nextPage);
+  }
+
+  return sprintValues;
+}
```

Comment:
this is easier to follow for me...```JS
  const nextPage = await pageSprints(teamId, startAt + 50);
  const sprintValues = sprintValues.concat(nextPage);

  return !sprints.isLast ? sprintValues || sprints.values;
}
```

Reply:
Much cleaner, thanks!

Reply:
Now that I read that again, it would break the paging since it checks `isLast` _after_ the call to `pageSprints`. It needs to check that before calling the API again.